docs/introduction/product-feeds.md

```diff
@@ -0,0 +1,33 @@
+# Product Feeds
+
+Product feeds are a way to periodically export information about your products for product search engines such as [Google Shopping](https://www.google.com/shopping).
+
+In order to allow easy installation and removal of product feeds, they are implemented in form of plugins ([see list of current implementations](https://github.com/search?q=topic%3Aproduct-feed+org%3Ashopsys)).
+
+## Where are the feeds?
+
+The exported files contain a random token generated during the application build, so their URL address is not easily guessed.
+You can see all installed product feeds along with the URLs of their export in the administration section *Marketing >  XML Feeds*.
+
+## When are they exported?
+
+Product feeds are usually exported using Cron modules.
+The Cron modules are already implemented and registered, all that's needed it to run the [`cron` phing target](phing-targets.md#cron) every 5 minutes on your server and Shopsys Framework takes care of the rest.
```

Comment:
~...all that's needed it...~ -> ...all that's needed **is**

Reply:
Fixed in 63c4c6255